api/client/src/test/java/org/projectnessie/client/http/BaseTestHttpClient.java

```diff
@@ -479,31 +480,107 @@ static Stream<Status> testHttpResponses() {
   }
 
   @ParameterizedTest
-  @MethodSource
+  @MethodSource("httpResponses")
   void testHttpResponses(Status status) throws Exception {
```

Comment:
why not just this?
```suggestion
  void httpResponses(Status status) throws Exception {
```

Reply:
Because the source is now being used for many test methods.